frontend/src/router.js

```diff
@@ -4,13 +4,21 @@ import VueRouter from 'vue-router';
 import HomePage from './components/HomePage.vue';
 import Login from './components/Login.vue';
 import Dashboard from './components/Dashboard.vue';
+import store from './store/index';
```

Comment:
If this is only one 'index.js' file in this directory, JS should look for by default:
```suggestion
import store from './store';
```